packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.doc.tsx

```diff
@@ -20,27 +20,27 @@ export const HoverCardPageProps: IDocPageProps = {
     'https://github.com/microsoft/fluentui/tree/master/packages/office-ui-fabric-react/src/components/HoverCard',
   examples: [
     {
-      title: 'Example 1: Expanding HoverCard wrapping an element',
+      title: 'Expanding HoverCard wrapping an element',
```

Comment:
One of the HoverCard examples needed cleanup, which made me notice that the docs did too...